src/components/select/select.js

```diff
@@ -370,29 +367,32 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
         if (angular.isString(disabled)) {
           disabled = true;
         }
+
         // Prevent click event being registered twice
         if (isDisabled !== undefined && isDisabled === disabled) {
           return;
         }
         isDisabled = disabled;
         if (disabled) {
-          element.attr({'tabindex': -1, 'aria-disabled': 'true'});
-          element.off('click', openSelect);
-          element.off('keydown', handleKeypress);
+          element
+            .attr({'aria-disabled': 'true'})
+            .removeAttr('tabindex')
+            .off('click', openSelect)
+            .off('keydown', handleKeypress);
         } else {
-          element.attr({'tabindex': attr.tabindex, 'aria-disabled': 'false'});
-          element.on('click', openSelect);
-          element.on('keydown', handleKeypress);
+          element
+            .attr({'tabindex': attr.tabindex || 0, 'aria-disabled': 'false'})
```

Comment:
OK, but because in line 191 we are setting a fallback value, we don't need to have "|| 0" in here, because "attr.tabindex" will be always truthy.